README.adoc

```diff
@@ -193,6 +193,19 @@ Preserve second fetch during initial checkout::
   This setting is only needed if there is a bug in the redundant fetch removal logic.
   If you enable this setting, please report a git plugin issue that describes why you needed to enable it.
 
+[[do-not-add-git-tag-action-to-jobs]]
+Do not add git tag action to jobs::
+
+  If checked, the git tag action will not be added to any builds that happen after the box is checked.
+  The checkbox only affects builds that happen after the box is checked.
++
+The git tag action allows a user to apply a tag to the git repository in the workspace based on the git commit used in the build applying the tag.
```

Comment:
This phrasing needs improvement, desperately.  The sentence is long and convoluted mess.
```suggestion
The git tag action allows a user to apply a tag to the git repository in the workspace based on the git commit used in the build applying the tag.
```

Reply:
```suggestion
The git tag action applies a tag to the git repository based on the commit used in the build applying the tag. 
```
Didn't really solve the problem, I assume `git` and `repository` are already known and hence removed them.